### PR TITLE
Added a functional buzzer:

### PR DESCRIPTION
**Pull request by MarcinSzymanek**
- Added a buzzertimer (ITimer) to UserInterface to keep track of buzzing
- Added buzzer interface: Toggle() and Stop()
- UserInterface now sets up the timer events in the constructor - requires ITimer and IBuzzer objects
- UserInterface now sets the buzzer timer at the end of cooking session
- When a door is opened, timer and buzzer are stopped
- Added simple buzzer tests for Toggle() and Stop()
- Program.cs updated with the necessary objects

### CODE REVIEW

File: MicrowaveOven/Microwave.Classes/Boundary/Timer.cs [hunk 1]
```diff
@@ -21,6 +21,7 @@ public Timer()
             timer.AutoReset = true;  // Repeatable timer
         }
 
+
```
Comment (CodeBloodedMama) [inline MicrowaveOven/Microwave.Classes/Boundary/Timer.cs:24]: måske unødvendig tastfejl?

File: MicrowaveOven/MicrowaveOven.sln [hunk 2]
```diff
@@ -1,16 +1,14 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30621.155
+# Visual Studio Version 17
```
Comment (CodeBloodedMama) [inline MicrowaveOven/MicrowaveOven.sln:3]: kører du ikke VS'22?

**Review by CodeBloodedMama — COMMENTED** [review]
Nice work. Easy to see the new feature implementation, good abstractionlayer with the interface

**Review by CodeBloodedMama — APPROVED** [review]

File: MicrowaveOven/Microwave.Classes/Boundary/Timer.cs (hunk 1 above)
Comment (MarcinSzymanek) [reply MicrowaveOven/Microwave.Classes/Boundary/Timer.cs:24]: Fat fingers!

File: MicrowaveOven/MicrowaveOven.sln (hunk 2 above)
Comment (MarcinSzymanek) [reply MicrowaveOven/MicrowaveOven.sln:3]: Jo, jeg ved ikke hvorfor den skriver alt det der? Måske er det fordi Frank gemte start solution i en gammel VS version. Startkoden / sol kommer fra Frank.

**Pull request merged**